backend/README.md

```diff
@@ -65,25 +65,140 @@ By completing these steps, the backend will be primed with the essential verifie
+
+To excute this example:
+```
+cargo run --example submit_ownership
+```
+
+Upon successful execution, you should see the message:
+```
+Ownership proofs are submitted successfully!
+```
+
+Reminder: This demonstration takes place in a test environment. In real-world production, always ensure that the Summa contract is correctly deployed on the target chain.
+
+### 3. Submit Proof of Solvency
+
+Before generate inclusion proof for every user of the current round, You should submit proof of sovlency to Summa contract. Currently, we made this as mandatory way to commit the root hash of the Merkle Sum Tree.
+
+Without this process, It seems the user may not trust to the inclusion proof for the round. becuase the `mst_root` is not published on contract. More specifically, it means that the `mst_root` is not correctly verified on the Summa contract.
+
+In this example, we'll guide you through the process of submitting a solvency proof using the Round to the Summa contract.
+The Round serves as the core of the backend in Summa, and we have briefly described it in the Components section.
+
+To initialize the Round, several parameters are required, including paths to specific CSV files (`assets.csv` and `entry_16.csv`), as well as a path to the ptau file (`ptau/hermez-raw-11`).
+
+The roles of these files are as follows:
+- `assets.csv`: This file is essential for calculating the total balance of assets for the solvency proof. Currently, only the CEX can generate this asset CSV file in its specific manner.
+
+- `entry_16.csv`: This file is used to build the Merkle sum tree, where each leaf element originates from sixteen entries in the CSV.
+X
+- `ptau/hermez-raw-11`: Contains the Powers of Tau trusted setup parameters, essential for constructing the zk circuits.
+
+By employing the round instance, the solvency proof is dispatched to the Summa contract using the `dispatch_solvency_proof` method.
+
+To excute this example:
+```
+cargo run --example submit_solvency
+```
+
+Upon successful execution, you will see the message:
+
+```
+ "Solvency proof is submitted successfully!"
 ```
 
-On successful execution, you'll observe a message indicating the verification outcome:
+### 4. Generating and Exporting Inclusion Proofs
+
+Assuming you are a CEX, let's say you've already committed the `solvency` and `ownership` proofs to the Summa contract. Now, you need to generate inclusion proofs for every user.
+
+In this example, we demonstrate how to generate and export user-specific inclusion proofs using the Round. This proof is crucial for users as it helps them validate the presence of specific elements within the Merkle sum tree, which forms a part of the solvency proof submitted.
+
+After generating the inclusion proof, end of example parts the inclusion proof is transformed into a JSON format, making it easily shareable.
+
+To excute this example:
```

Comment:
Can probably be fixed everywhere at once by running find&replace

Reply:
Fixed at [eacfe7d](https://github.com/summa-dev/summa-solvency/pull/159/commits/eacfe7df0542f35ab9367cc34ad8a298b0cdda5d)